### ISSUE TITLE
Please allow devs to add custom placeholders, or none placeholder

### ISSUE THREAD

**Issue by wemersonrv** [techouse/intl-date-time#6]
Hello.

Please allow devs to add custom placeholders, or none placeholder. See my issue:

Here on Brazil we say format as (DD/MM/AAAA) not (DD/MM/YYYY)... obviously when coding, will use the right compatible pattern `DD/MM/YYYY` but when show it to users in frontend, need to inform `DD/MM/AAAA` so they will understand. But the field not allow custom placeholder and add the mask as it... 

So i add a `->help('DD/MM/AAAA')` to give a tip to users, but tis is weird... because it renders both patterns:

```php
IntlDateTime::make('Nascimento', 'nascimento')
            ->dateFormat('DD/MM/YYYY')
            ->help('DD/MM/AAAA')
```
See image below:
![intl2](https://user-images.githubusercontent.com/2028673/58024856-8773e980-7ae9-11e9-9e6e-9a6ed8336505.png)

**Comment by techouse** [comment]
The placeholder is actually tied to the mask. Will look into it.

**Comment by techouse** [comment]
Available from [v1.1.2](https://github.com/techouse/intl-date-time/commit/7204baab98d2f4cc71554f9c1437ffc03ca2531f) onward.
